debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleBlobDataTypesIT.java

```diff
@@ -1401,6 +1401,242 @@ public void shouldStreamBlobWithUnavailableColumnValuePlaceholder() throws Excep
         }
     }
 
+    @Test
+    @FixFor("DBZ-4366")
+    public void shouldStreamBlobsWrittenInChunkedMode() throws Exception {
```

Comment:
Could a similar test be included in `OracleClobDataTypesIT` for `CLOB` and `NCLOB` so we can verify and have test coverage for those in case there are any variants in behavior?